config/izum_vega.py

```diff
@@ -0,0 +1,119 @@
+from os import environ
+username = environ.get('USER')
+
+# This is an example configuration file
+site_configuration = {
+    'general': [
+        {
+            'remote_detect': True,
+        }
+    ],
+    'systems': [
+        {
+            'name': 'vega',
+            'descr': 'Vega, a EuroHPC JU system',
+            'modules_system': 'lmod',
+            'hostnames': ['vglogin*','cn*','gn*'],
+            'stagedir': f'reframe_runs/staging',
+            'outputdir': f'reframe_runs/output',
+            'partitions': [
+                {
+                    'name': 'cpu',
+                    'scheduler': 'slurm',
+                    'prepare_cmds': [
+                        'source /cvmfs/pilot.eessi-hpc.org/latest/init/bash',
```

Comment:
Should we name this partition `cpu_eessi`, and then also have a "native" `cpu` partition?

Or maybe even a separate configuration file like `izum_vega_eessi.py`, since that requires extra work before running the `reframe` command (clean out `$MODULEPATH` + `source` EESSI init script)

Reply:
I think we also discussed this in the meeting, but I'd be up for a separate config. The easiest way to run both EESSI and native module tests would be running two separet commands:

- If you want to run on the EESSI modules, you have to run the `reframe` command in an environment where EESSI is sourced, and with a config that also sources it
- If you want to run on the native modules, you have to run the `reframe` command in an environment where EESSI is _not_ sourced, and with a config that also does _not_ source it

Trying to do both in a single run is not possible: we could specify e.g. two CPU partitions with different `prepare_cmds`, but a single `reframe` run in an environment that has both the EESSI and native modules available would test _all_ those modules on _both_ those CPU partitions.